Include/cpython/unicodeobject.h

```diff
@@ -67,14 +71,6 @@
 /* low surrogate = bottom 10 bits added to DC00 */
 #define Py_UNICODE_LOW_SURROGATE(ch) (0xDC00 + ((ch) & 0x3FF))
 
-/* Check if substring matches at given offset.  The offset must be
-   valid, and the substring must not be empty. */
-
-#define Py_UNICODE_MATCH(string, offset, substring) \
```

Comment:
Please documented the removed function at https://docs.python.org/dev/whatsnew/3.10.html#id3 since it's part of the public C API. It would be even better if you can suggest a solution: replace it with PyUnicode_Tailmatch().